README.md

```diff
@@ -45,7 +45,7 @@ This launcher also supports other game client, including:
 * ~~Zenless Zone Zero~~ (Waiting for Official Release)
 
 > **Note**:
-> Some features such as Game Data Repair and Preloading are available for Genshin Impact, but others such as Game Settings modification and Caches Download are not available and presently unsupported.
+> Some features such as Game Data Repair and Preloading are available for other games, but more advanced features such as Game Settings modification and Caches Download are currently not available and unsupported.
```

Comment:
I wouldn't call modifying game settings and downloading update caches "advanced features", what's your reasoning for it? The other changes are fine.

Reply:
I call it `more advanced features` as it goes beyond what the official launcher does (Also the fact that in Genshin you had to go through a mountain of hex formatted json that has an XML file on it for some reason LOL). Tho honestly, I just couldn't think of another word to separate those features for the base ones (Downloading, Installing, Repairing) :TeriDerp:

Reply:
I see, we can keep your changes for this one then. Thanks!